src/components/useFetchRecipes.js

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from "react";
+
+// Returns a valid query parameter string depending on the ingredients state
+const createIngredientQueryParam = (ingredients) => {
+  let res = "";
+  for (let i in ingredients) {
```

Comment:
do you mean "let i of ingredients"

javascript's for-in loop will iterate over enumerable string properties while the for-of loop will iterate over the items in an iterable: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#difference_between_for...of_and_for...in

Reply:
changed to of